man/ocitools-generate.1.md

```diff
@@ -234,6 +234,45 @@ inside of the container.
   The special *PATH* `host` removes any existing UTS namespace from
   the configuration.
 
+**--cpu-shares**=CPUSHARES
+  Sets the CPU shares (relative weight (ratio) vs. other cgroups with cpu shares).
```

Comment:
Instead of having two descriptions to discuss, I'd rather just copy the runtime-spec wording verbatim here.  For this setting, [that's][1]:

> Specifies a relative share of CPU time available to the tasks in a cgroup.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#cpu